packages/components/organisms/f-header/stories/header.stories.js

```diff
@@ -49,6 +49,12 @@ export const HeaderComponent = () => ({
         },
         showSkipLink: {
             default: boolean('Show skip link', true)
+        },
+        showCustomNavLinks: {
```

Comment:
Not sure if it's a bit confusing to have this control in storybook when it's not an actual prop? If you think it's worthwhile having here, I'd probably leave a comment just to say that the knob is only there for storybook to aid in showing and hiding an example of the links, as I'm guessing that's why its there?

Reply:
Have updated the story to use controls rather than knobs, which makes (or will make) it easier to pass complex values and allow for proper descriptions.